index.md

```diff
@@ -310,6 +310,7 @@ color by default via `NO_COLOR`.
 | [Radare](https://www.radare.org/r/) | `radare2 -e scr.color=0` |
 | [RSpec](http://rspec.info/) | `export SPEC_OPTS=--no-color` |
 | [Ruby (tests)](https://www.ruby-lang.org/) | `export RUBY_TESTOPTS=--color=never` |
+| [Terraform](https://www.terraform.io/) | `terraform [main command] -no-color` ([Docs](https://developer.hashicorp.com/terraform/cli/commands/plan#no-color) \| [Open Issue](https://github.com/hashicorp/terraform/issues/23708)) |
```

Comment:
I believe it should have two hyphens; in this PR, there is only one (i.e., it's currently `-no-color`, but should be `--no-color`).

Reply:
No, Terraform uses a single `-no-color` for this argument.